### PR TITLE
fix(VListitem): disable mouse click in animating menu

### LINKED ISSUE #12021: [Bug Report] Double click on dropdown menu with a disabled list-item triggers click event on that list-item
### Environment
**Vuetify Version:** 2.3.1
**Vue Version:** 2.6.11
**Browsers:** Safari 13.1.1
**OS:** Mac OS 10.14.6

### Steps to reproduce
Make a double-click on the bottom half of "Dropdown" button.

### Expected Behavior
No click event is trigger. No click listener of the disabled list-item is called.

### Actual Behavior
onClick handler of the first contained item is called despite that it is disabled.

### Reproduction Link
<a href="https://codepen.io/sbarlabanov/pen/dyMPeMp" target="_blank">https://codepen.io/sbarlabanov/pen/dyMPeMp</a>

### Other comments
This happens when the menu items are shown over the button (like in the example), which may always happen depending on the window size and layout. Seems like the second click is captured by the list-item despite "pointer-events: none".

### PR DESCRIPTION
**Pull request by YipingRuan**
fix #12021

## Markup:

<details>

```vue
<template>
  <div id="app">
    <v-app id="inspire">
      <div class="text-center">
        <v-menu top>
          <template v-slot:activator="{ on, attrs }">
            <v-btn
              color="primary"
              dark
              v-bind="attrs"
              v-on="on"
            >
              Dropdown
            </v-btn>
          </template>

          <v-list>
            <v-list-item
              v-for="(item, index) in items"
              :key="index"
              disabled
              @click="onClick($event)"
            >
              <v-list-item-title>{{ item.title }}</v-list-item-title>
            </v-list-item>
          </v-list>
        </v-menu>
      </div>
    </v-app>
  </div>
</template>

<script>
  export default {
    data: () => ({
      items: [
        { title: 'Click Me Disabled' },
        { title: 'Click Me' },
        { title: 'Click Me' },
        { title: 'Click Me 2' },
      ],
    }),
    methods: {
      onClick (d) {
        alert('ups, clicked on a disabled item')
        console.log(d)
      },
    },
  }
</script>

```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VList/VListItem.sass [hunk 1]
```diff
@@ -4,6 +4,7 @@
 +theme(v-list-item) using ($material)
   &--disabled
     color: map-deep-get($material, 'text', 'disabled')
+    pointer-events: none
```
Comment (KaelWD) [inline packages/vuetify/src/components/VList/VListItem.sass:7]: This is already set on line 38. Non-theme styles shouldn't go in the theme section as they'll be duplicated.

Comment (YipingRuan) [reply packages/vuetify/src/components/VList/VListItem.sass:7]: So line 38 was override by enter-to at Vmenu.sass line:48. Should I remove this one instead?

image

Comment (KaelWD) [reply packages/vuetify/src/components/VList/VListItem.sass:7]: Uh yeah that definitely shouldn't be there, I thought it was a browser bug. 

8cc1c719a4ee18b195db2b42182a5700fda72291

Comment (YipingRuan) [reply packages/vuetify/src/components/VList/VListItem.sass]: Updated, please verify.

**Pull request merged**